plugins/packages/couchdb/lib/manifest.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ToolJet/ToolJet/develop/plugins/schemas/manifest.schema.json",
+  "title": "CouchDB datasource",
+  "description": "A schema defining CouchDB datasource",
+  "type": "database",
+  "source": {
+    "name": "CouchDB",
+    "kind": "couchdb",
+    "exposedVariables": {
+      "isLoading": false,
+      "data": {},
+      "rawData": {}
+    },
+    "options": {
+      "username": {
+        "type": "string"
```

Comment:
option types are not used anymore, but should keep the encryption issue, 
```
"options": {
   "password": {
     "encrypted": true
   },
}
```
this is enough